earthfile2llb/listener.go

```diff
@@ -962,6 +968,16 @@ func (ssf *StringSliceFlag) Set(arg string) error {
 	return nil
 }
 
+var shellNameStringRegexp = regexp.MustCompile("^[a-zA-Z_]+[a-zA-Z0-9_]*")
+
+func checkEnvVarName(str string) error {
+	matchedString := shellNameStringRegexp.FindString(str)
```

Comment:
Should rather use [Match](https://golang.org/pkg/regexp/#Regexp.Match) for this

Reply:
Any reason to use Match instead of [MatchString](https://golang.org/pkg/regexp/#Regexp.MatchString)?

Reply:
You are right indeed. MatchString is correct.